web3.md

```diff
@@ -1728,7 +1731,7 @@ Timestamp Calls
  // -----------------------------------
     rule <k> #firefly_setTime => #rpcResponseSuccess(true) ... </k>
          <params> [ TIME:String, .JSONs ] </params>
-         <timestamp> _ => #parseHexWord( TIME ) </timestamp>
+         <timeDiff> _ => #parseHexWord( TIME ) -Int #time() </timeDiff>
```

Comment:
Should we also be adjusting `<timestamp>` here?

Reply:
Nope. `<timestamp>` is just updated by `<timeDiff>` for tx executions now.

Reply:
And also when `evm_mine` is called. Basically whenever the blockchain is going to get updated.